tests/test_display.py

```diff
@@ -14,6 +14,22 @@
 import py_trees.display as display
 import xml.etree.ElementTree
 
+##############################################################################
+# Helpers
+##############################################################################
+
+
+def assert_banner():
```

Comment:
BTW probably more meaningful if these methods were prefixed with print_ to indicate their purpose.

Reply:
I've got the same calls across the other `py_trees` repositories. I'd have to change them all to be consistent. Would prefer to leave as is as it's a nice-to-have only.

Reply:
Done.